utils/dev-tool/src/utils.rs

```diff
@@ -0,0 +1,168 @@
+use serde::{Deserialize, Serialize};
+use std::env::VarError;
+use std::fs::File;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use std::{env, fs};
+
+#[macro_export]
+macro_rules! panic_if_unsuccessful {
```

Comment:
This is an interesting name for what is essentially an assert macro. I propose `assert_successful` or something along those lines instead.